### DISCUSSION TITLE
VPN mode

### DISCUSSION THREAD

**Discussion by Supersimmons** [guardianproject/orbot-android#759] [Q&A] [answered]
hello, why some apps don't open if you tap on them while some others do.  For example Firefox works but other apps like facebook or app bank don't. 
Until recently all apps opened just fine. Has something changed?

Thanks
Simone

**Comment by n8fr8** [comment, marked as answer]
Not all apps support the "View" option for the https://check.torproject.org link we are trying to send to the app. The tapping really only makes sense for browsers. Sorry that it isn't clear what the tap option is for!

**Reply by Supersimmons** [reply]
I'm sorry but I explained myself badly.
I meant that when I try to open an app in vpn mode other than firefox, it doesn't open.
Until recently I had no problems.
In this example neither facebook nor amazon work, but firefox does.
![Screenshot](https://user-images.githubusercontent.com/1744667/203531301-87c297ed-270c-4dce-b61d-ee2c9f590467.jpg)
